sig-windows/annual-report-2022.md

```diff
@@ -4,17 +4,18 @@
 
 1. What work did the SIG do this year that should be highlighted?
```

Comment:
Should we add unit test improvements and the CRI KEP stuff?

Reply:
Oh I see the unit test stuff below

Reply:
I'll add the CRI KEP